### PR TITLE
Lavaland Overhaul: Part 6 - Mining Medics

### PR DESCRIPTION
**Pull request by Super3222**
https://github.com/yogstation13/yogstation/pull/1773

#### Intent of your Pull Request

Mining Medics are getting a BIG upgrade, and this is mostly based on what I've observed of players in the game (and have went through in game myself).

To start of - MM's lockers will spawn with a new durable EMT jacket that can carry big oxygen tanks. Their lockers will also spawn with a mini fire extinguisher for obvious reasons.

Mining Medics spawn with medical webbing - which is a fusion between an actual webbing and a medical belt. It can carry your usual mining gear, pickaxes, scanners, and knives, but it also supports ointment, bruise packs, and more.

Their lockers will also have 3 new things -
Two Hyposprays - One carrying Bicardrine and another carrying Kelatone.

The ATV's action button switches sprites based on whether the light is on or off.

I've also re-added the minimap with some tweaks in an attempt to mask ruins.

Mining Medics will additionally lose their Mining EVA access. In other words, they can't leave the mining station.

##### Changelog

:cl:
tweak: Mining Medics can no longer leave the mining station.
rscadd: Mining Medics have more durable EMT jackets that can carry large oxygen tanks.
rscadd: Mining Medic's lockers spawn with a mini fire extinguisher.
rscadd: Mining Medics spawn with a medical webbing - a fusion between an actual webbing and a medical belt.
rscadd: Mining Medic's lockers spawn with two new hyposprays. One containing kelatone and the other containing bicardine.
tweak: The ATV action button sprites show whether it's on or off.
experimental: Made it impossible to tell where ruins are on the crew monitor console. Also re-enabled the lavaland minimap.
/:cl:

### CODE REVIEW

**Comment by Nickvr628** [comment]
+1

**Comment by BOBsHERE** [comment]
the days of deathchems + mining medic are back 0.o

**Comment by Keekenox** [comment]
Hypospray! Noooo! Also mining medics will ALWAYS just be an advanced miner who spawns with meds to heal THEMSELVES

**Comment by Time-Green** [comment]
I agree with keek. People go mining medic, simply to grab all the medicine and use it for themselves. Some dude Noah Cooper does this every round

**Comment by Zostroll** [comment]
The mining medic only had the hypo before because of hardsuits. I am against readding the hypo.

**Comment by Super3222** [comment]
@puremanbird don't get your murderboner up yet!
@Keekenox Don't worry these are the nerfed versions of the hypospray. They cannot be refilled. I see what you mean about using those meds on themselves. Hmm...
@Zostroll That's a pretty good point.
@Time-Green I've actually seen him in action. Not too fond of it given that he usually steals some of the miners' KA's. This PR will remove mining medic's access to miner lockers.

I'll take some more time on this to develop them. Thanks for your comments everyone.

**Comment by ShadowDeath6** [comment]
>Bicardrine

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 1]
```diff
@@ -155,3 +155,21 @@
...
+	desc = "A quick-mix capital combat injector loaded with miximone."
+	amount_per_transfer_from_this = 5
+	icon_state = "combat_hypo"
+	volume = 50
+	list_reagents = list("bicaridine" = 50)
+	flags = 0
+
+/obj/item/weapon/reagent_containers/hypospray/derm
+	name = "QMC Kelotane Injector"
+	desc = "A quick-mix capital combat injector loaded with dermaline."
```
Comment (ShadowDeath6) [inline code/modules/reagents/reagent_containers/hypospray.dm]: Dermaline != kelotane

**Comment by Super3222** [comment]
Alright guys. I'm thinking about completely limiting the medics mobility to the mining station. Can I gather a few thoughts on this? Removing their EVA access has to happen so they can't steal explorer jumpsuits.

I could also possibly install a shutter somewhere in the mining base for the mining medic to drive their ATV through, and venture into lavaland in search for miners. Of course, after limiting a multitude of their equipment and access to weapons.

**Comment by ShadowDeath6** [comment]
>has one shutter
it has lava in front of it

**Comment by Super3222** [comment]
I just came up with a much better idea. I'm still going to limit a lot of what mining medics are able to do right now.

**Comment by Super3222** [comment]
I'm limiting mining medics to the mining station for now. In Lavaland Overhaul Part 8 I'll bring back the jaunter beacons, and they'll have a higher purpose.

File: code/modules/reagents/reagent_containers/hypospray.dm (hunk 1 above)
Comment (Cruix) [reply code/modules/reagents/reagent_containers/hypospray.dm]: Still says dermaline

**Pull request merged**